src/js/entities-service/entities/actions.js

```diff
@@ -22,21 +30,13 @@ const _Model = BaseModel.extend({
 
     return forms.at(0);
   },
-  getClinician() {
-    const clinicianId = this.get('_clinician');
-    if (!clinicianId) return;
-    return Radio.request('entities', 'clinicians:model', clinicianId);
-  },
   getPatient() {
     return Radio.request('entities', 'patients:model', this.get('_patient'));
   },
-  getRole() {
-    const roleId = this.get('_role');
-    if (!roleId) return;
-    return Radio.request('entities', 'roles:model', roleId);
-  },
   getOwner() {
-    return this.getClinician() || this.getRole();
+    const owner = this.get('_owner');
+    if (!owner || !owner.id) return;
```

Comment:
are both of these cases possible?  I think only `!owner` should be an issue... but I'm not 100% sure what happens with new actions.  It could be this is just supporting the `toRelation` below and won't be needed if that's fixed.  In fact I don't think actions can ever have a null owner.  program-actions can, but actions should 100% of the time have an `_owner`

Reply:
This was just an overly paranoid check. But yeah, there's no way for id to be empty. It's always just `owner` that is null.